libia2/include/ia2.h

```diff
@@ -36,6 +36,12 @@
 
 #endif
 
+static uint32_t ia2_get_pkey() {
```

Comment:
I would rather this were named `ia2_get_pkru`. The terminology convention we're following here is that there are 16 pkeys controlled by the one pkru register, and its value determines which set of pkeys the process has.

Reply:
We have several definitions of this as `rdpkru` in tests that should be changed to use this definition as a follow-up.